app/models/trip.rb

```diff
@@ -1,5 +1,5 @@
 class Trip < ApplicationRecord
-  belongs_to :user
+  belongs_to :user, optional: true
```

Comment:
Why is this optional? Why would we have trips that are not assigned to users?

Reply:
exactly. otherwise it would fail